### DISCUSSION TITLE
Does clustered broker pods share queue messages between themselves so that consumer can get msg from any broker irrespective of where producer sent

### DISCUSSION THREAD

**Discussion by Laveshsp** [arkmq-org/arkmq-org-broker-operator#1067] [Q&A]
Requirement: I want to set up multiple broker pods having a single external endpoint.

So, want to know whether in the clustered broker pod (or any other artemis topology) set up, if we send queue message to any one broker pod in cluster, will that queue message get reflected on other broker pods too automatically so that they can be consumed from any other broker pod later by the consumers.

Does Artemis support this? If yes, what setup is best suited for this? If any docs for reference, that would be great

**Comment by brusdev** [comment]
By default, the broker pods related to a single ActiveMQ Artemis CR are connected in a cluster and the message distribution is enabled. You need to enable redistribution to if you need to connect to any broker pod of the cluster to send or receive messages, see https://activemq.apache.org/components/artemis/documentation/latest/clusters.html#message-redistribution

**Reply by Laveshsp** [reply]
Thanks for pointing me to the doc. Let me try this setup and see how it behaves.
